### PR TITLE
Support for static RecipientDetails for certain shortcodes.

### PR DESCRIPTION
**Pull request by moxie0**
// FREEBIE

### CODE REVIEW

File: src/org/thoughtcrime/securesms/recipients/RecipientProvider.java [hunk 1]
```diff
@@ -143,7 +151,8 @@ public RecipientDetails call() throws Exception {
         cursor.close();
     }
 
-    return new RecipientDetails(null, number, null, ContactPhotoFactory.getDefaultContactPhoto(null), color);
+    if (STATIC_DETAILS.containsKey(number)) return STATIC_DETAILS.get(number);
```
Comment (mcginty) [inline src/org/thoughtcrime/securesms/recipients/RecipientProvider.java]: i feel like the user contact db should override whatever we hardcode

Comment (moxie0) [inline src/org/thoughtcrime/securesms/recipients/RecipientProvider.java]: it does

Comment (mcginty) [inline src/org/thoughtcrime/securesms/recipients/RecipientProvider.java]: ah i misread, this is just what it fills in before resolving in the contacts db.

yeah this lgtm

**Comment by moxie0** [comment]
in 2.24.0

**Pull request closed without merging**